sdcm/cluster_docker.py

```diff
@@ -307,6 +309,7 @@ def node_setup(self, node, verbose=False, timeout=3600):
         self.clean_replacement_node_ip(node)
 
     @staticmethod
+    @retrying(n=5, sleep_time=3, allowed_exceptions=(SSHException, ))
```

Comment:
I am not sure this will help since we retry all network exceptions in `sdcm.remote.RemoteCmdRunner.run`

Reply:
maybe it would be good enough, to change to do this check with localrunner ? since this value should be the same on the worker, we don't need doing this via ssh inside the docker.

Reply:
> 
> 
> maybe it would be good enough, to change to do this check with localrunner ? since this value should be the same on the worker, we don't need doing this via ssh inside the docker.

Yeah, I thought about this way initially, but I think it's more future-proof to check it from inside a container. In this case we don't need to make assumption that our localhost == our Docker host. In current design I prefer to stay with per-node check.

Reply:
> 
> 
> I am not sure this will help since we retry all network exceptions in `sdcm.remote.RemoteCmdRunner.run`
Great. Looks like, I can remove retry and add "No existing session" string to `is_error_retriable()` function.